api/dependencies/file.py

```
@@ -282,160 +318,169 @@ def get_file_validator(max_size: int=None, allowed_types: List[str]=None,
         if isinstance(allowed_types, str):
             allowed_types = [allowed_types]
 
-    async def validator(file: UploadFile=File(...)) ->Tuple[UploadFile,
-        FileAnalysisReport]:
+    async def validator(file: UploadFile = File(...)) -> Tuple[UploadFile, FileAnalysisReport]:
```

Pylint (3.3.8)

[refactor] Too many branches (14/12)

(R0912)